src/utilities/activity-utils.ts

```diff
@@ -133,8 +133,10 @@ export const getLinkedPluginEmbeddable = (page: Page, id: string) => {
 
 export const setAppBackgroundImage = (backgroundImageUrl?: string) => {
   const gradient = "linear-gradient(to bottom, #fff, rgba(255,255,255,0), rgba(255,255,255,0))";
-  const el = document.querySelector(".app") as HTMLElement;
-  el?.style.setProperty("background-image", backgroundImageUrl || gradient);
+  const el = document.querySelector("#app") as HTMLElement;
```

Comment:
@mklewandowski @scytacki I made another small change to line 136 that I think should be harmless, but please take a quick look and let me know if you see any issues.

When the background image was set on the `.app` element, the image wouldn't always reach the bottom of the browser window.  So I changed it to apply the background to the `#app` element.